docs/releases/1.11.4.txt

```diff
@@ -35,3 +35,6 @@ Bugfixes
   ``checkbox_name``, ``checkbox_id``, ``is_initial``, ``input_text``,
   ``initial_text``, and ``clear_checkbox_label`` are now attributes of
   ``widget`` rather than appearing in the top-level context.
+
+* Fixed a regression where `GenericRelations` to proxy models raised an
```

Comment:
You need to use double backticks to have an effect on the rendered docs. Suggestion:
```
* Fixed queryset crash when using a ``GenericRelation`` to a proxy model
  (:ticket:`28418`).
```